cltk/corpus/latin/readers.py

```diff
@@ -0,0 +1,84 @@
+"""Readers for Latin language corpora"""
+
+__author__ = ['Patrick J. Burns <patrick@diyclassics.org>']
+__license__ = 'MIT License. See LICENSE.'
+
+import os.path
+
+from nltk.corpus.reader.plaintext import PlaintextCorpusReader
+from nltk.data import LazyLoader
+
+from cltk.tokenize.sentence import TokenizeSentence
+from cltk.tokenize.word import WordTokenizer
+from cltk.utils.cltk_logger import logger
+from cltk.corpus.utils.importer import CorpusImporter
+
+
+# Set up models
+home = os.path.expanduser('~')
+cltk_path = os.path.join(home, 'cltk_data')
+corpus_directory = cltk_path + '/latin/text/latin_text_latin_library'
+
+word_tokenizer = WordTokenizer('latin')
+
+# Prepare sentence tokenizer
+# Incorporate way to fall back to NLTK English sentence tokenizer?
+def get_latin_sent_tokenizer():
+    """
+    Function that gets Latin sentence tokenizer
+    """
+    try:
+        latin_sent_tokenizer = TokenizeSentence('latin')
+        #logger.debug("Loaded the NLTK's PlaintextCorpusReader.")
+    except (IOError, AssertionError) as io_err:
+        logger.error("Couldn't find Latin models. Please import.")
+        print("Latin model needs to be installed to use this tokenizer. Please import and try again.")
+        install = input('Latin model not found. Import? [Y/n] ')
```

Comment:
I think we need separate methods for install and for fetching the sent_tokenizer;
in typical API design it's best to not assume that the library will have a person sitting at a terminal to provide input, paradoxically, better to throw an error and let the user run a specific install script, just maybe advise that in the error message.